framegear/components/Frame/Frame.tsx

```diff
@@ -14,40 +15,14 @@ export function Frame() {
 
   const latestFrame = results[results.length - 1];
 
-  if (!latestFrame.isValid) {
```

Comment:
Removed this because right now it's more useful to fully render the frame even if it is not 100% correct